app/src/main/java/org/simple/clinic/home/overdue/search/OverdueSearchEvent.kt

```diff
@@ -21,3 +21,5 @@ data class OverduePatientClicked(val patientUuid: UUID) : OverdueSearchEvent()
 data class CallPatientClicked(val patientUuid: UUID) : OverdueSearchEvent()
 
 data class OverdueSearchHistoryClicked(val searchQuery: String) : OverdueSearchEvent()
+
+data class OverdueSearchLoadStateChanged(val overdueSearchProgressState: OverdueSearchProgressState) : OverdueSearchEvent()
```

Comment:
If we are naming the enum as `OverdueSearchProgressState`  then let's name the event as `OverdueSearchProgressStateChanged` instead?

Reply:
So, this event is triggered when the paging adapter load state is changed. So, we are following that naming convention. Similar to what we did in instant search screen.

Reply:
umm okay